addons/website_slides/views/slide_question_views.xml

```diff
@@ -5,6 +5,13 @@
         <field name="model">slide.question</field>
         <field name="arch" type="xml">
             <form string="Quiz">
+                <div attrs="{'invisible': [('answers_validation_error', '=', '')]}">
```

Comment:
Can you check if we shouldn't set it to `False` instead? Especially if it is used like this, it could be more "usual" and robust on `NULL`?

Reply:
`answers_validation_error` is currently a computed `Char field`. Assessing whether it is `False` does not provide the expected behaviour on the frontend. Shall I turn `answers_validation_error` into a `Boolean field`?